.github/workflows/onPushToMain.yml

```diff
@@ -9,36 +9,32 @@ jobs:
     uses: salesforcecli/github-workflows/.github/workflows/unitTest.yml@main
   release:
     runs-on: ubuntu-latest
-    container:
-      image: node:lts
     needs:
     - tests
     steps:
-    - uses: actions/checkout@v3
-      with:
-        token: ${{ secrets.IDEE_GH_TOKEN }}
-    - uses: ./.github/actions/gitConfig
```

Comment:
The other job doesn't use this, so I removed the call to the gitConfig as well. I don't think anything else is using it, so I could remove this section too, or keep it in just in case.

Reply:
better to be gone if it can be 👍